core/src/main/java/org/bitcoinj/core/Wallet.java

```diff
@@ -3225,6 +3226,82 @@ private void checkBalanceFuturesLocked(@Nullable Coin avail) {
         }
     }
 
+    /**
+     * Returns the amount of bitcoin ever received via output. THIS IS NOT THE BALANCE! If an output spends from a
```

Comment:
I don't think we need to scream this (uppercase letters). If you want to emphasize something, use HTML \<b\> or \<i\> tags.